plugin/src/main/java/org/opensearch/ml/model/MLModelCacheHelper.java

```diff
@@ -299,7 +300,7 @@ public synchronized void setIsAutoDeploying(String modelId, Boolean isModelAutoD
      */
     public boolean isAutoDeploying(String modelId) {
         MLModelCache modelCache = modelCaches.get(modelId);
-        return modelCache != null && modelCache.getIsAutoDeploying() == true;
+        return modelCache != null && modelCache.getIsAutoDeploying() != null && BooleanUtils.isTrue(modelCache.getIsAutoDeploying());
```

Comment:
`BooleanUtils.isTrue` already covers null case, do we really need to add `modelCache.getIsAutoDeploying() != null` ?

Reply:
Checked and verified again. Shouldn't need so removed it already.